src/biip/gs1/_messages.py

```diff
@@ -4,10 +4,12 @@
 
 from dataclasses import dataclass
 from typing import Iterable, List, Optional, Union
+import re
 
 from biip import ParseError
 from biip.gs1 import DEFAULT_SEPARATOR_CHARS, GS1ApplicationIdentifier, GS1ElementString
 from biip.gtin import RcnRegion
+import biip.gs1._application_identifiers
```

Comment:
This should make isort happy and make the code shorter:

```suggestion
from biip import ParseError
from biip.gs1 import DEFAULT_SEPARATOR_CHARS, GS1ApplicationIdentifier, GS1ElementString
from biip.gs1._application_identifiers import _GS1_APPLICATION_IDENTIFIERS
from biip.gtin import RcnRegion
```